src/lib.rs

```diff
@@ -10,3 +10,6 @@ pub mod rpc_client;
 pub mod types;
 pub mod util;
 pub mod validate;
+
+#[cfg(test)]
+mod arbitrary;
```

Comment:
A cool name for a module!   :slightly_smiling_face:

Reply:
haha that's somewhat common for strategies related to `proptest`